### ISSUE TITLE
tag a new release?

### ISSUE THREAD

**Issue by simonbyrne** [tpapp/DynamicHMC.jl#37]
I'm getting an error on the release which doesn't appear on master, so would be good to have a new version.

**Comment by tpapp** [comment]
Please paste in the error, and the relevant `Manifest.toml`.

**Comment by simonbyrne** [comment]
```
MCMC, adapting ϵ (400 steps)
step 100 (of 400), 0.15 s/step
step 200 (of 400), 0.13 s/step
step 300 (of 400), 0.13 s/step
step 400 (of 400), 0.12 s/step
0.12 s/step ...done
MCMC, adapting ϵ (50 steps)
ERROR: LoadError: ArgumentError: 0 ≤ a must hold. Got
0 => 0.0
a => NaN
Stacktrace:
 [1] adapt_stepsize(::DynamicHMC.DualAveragingParameters{Float64}, ::DynamicHMC.DualAveragingAdaptation{Float64}, ::Float64) at /home/simonbyrne/.julia/packages/ArgCheck/BUMkA/src/checks.jl:165
...
```

The Manifest.toml is pretty big (it's part of a large project):
```
[[DynamicHMC]]
deps = ["ArgCheck", "DataStructures", "DocStringExtensions", "LinearAlgebra", "LogDensityProblems", "Parameters", "Pkg", "Random", "Statistics", "StatsFuns", "Test", "TransformVariables"]
git-tree-sha1 = "10315592b6d6c2f5b0762b337c17898544b79902"
uuid = "bbc10e6e-7c05-544b-b16e-64fede858acb"
version = "1.0.2"
```

As I said, I don't see the error on master (possibly fixed by #33)?

**Comment by tpapp** [comment]
I just tagged a release, thanks for the report. Will close the issue when it trickles through the registry.
